personio-personnel-data-api-oa3.yaml

```diff
@@ -3961,6 +3963,10 @@ components:
               type: string
               nullable: true
               example: offsite_work
+            legacy_category:
+              type: string
```

Comment:
Hi! Just a question. Is `legacy_category` always a string, or can it be an `ENUM`?

Reply:
it can be an enum, yes. Currently, in the DB we have `category` and `categoryV2`.  The current state of the PAPI is that the category attribute in the response ->  `category`, after the release:
category  ->  `categoryV2`,
legacy_category -> `category`


So, should we document both fields as enums, or since nothing much has changed, we should keep them as they were?

Reply:
This documentation is intended for customers, so if the expected values are already known and set, then using an ENUM would make things more clear for them. If the values can change, then we can leave it as a string. What do you think?